_mobile/ios.md

```diff
@@ -255,11 +255,11 @@ To use the custom built libraries the project, replace `#import <LibTorch-Lite/L
 Finally, disable bitcode for your target by selecting the Build Settings, searching for **Enable Bitcode**, and set the value to **No**.
 
 ## Using the Nightly PyTorch iOS Libraries in CocoaPods
-If you want to try out the latest features added to PyTorch iOS, you can use the `LibTorch-Lite-Nightly` pod in your `Podfile`: 
+If you want to try out the latest features added to PyTorch iOS, you can use the `LibTorch-Lite-Nightly` pod in your `Podfile`, it includes the nightly built libraries: 
 ```
 pod 'LibTorch-Lite-Nightly'
 ```
-It includes the nightly built libraries. But be aware you may need to build the model used on mobile in the latest PyTorch - using either the latest PyTorch code or a quick nightly install with commands like `pip install --pre torch torchvision -f https://download.pytorch.org/whl/nightly/cpu/torch_nightly.html` - to avoid possible model version mismatch errors when running the model on mobile. Note that if you wish to update the nightly pod to the newer one, you should run `pod deintegrate` to clear the local cache, and then run `pod install` to get the latest version.
+And then run `pod install` to add it to your project. If you wish to update the nightly pod to the newer one, you can run `pod update` to get the latest version.  But be aware you may need to build the model used on mobile in the latest PyTorch - using either the latest PyTorch code or a quick nightly install with commands like `pip install --pre torch torchvision -f https://download.pytorch.org/whl/nightly/cpu/torch_nightly.html` - to avoid possible model version mismatch errors when running the model on mobile. 
```

Comment:
@hanton Have you tested if the nightly works well with the torchvision nightly? cc @husthyc

Reply:
Thanks for the heads up! Tested the nightly pod by going through [(PROTOTYPE) USE IOS GPU IN PYTORCH](https://pytorch.org/tutorials/prototype/ios_gpu_workflow.html). Is this what you mean?

Reply:
> Thanks for the heads up! Tested the nightly pod by going through [(PROTOTYPE) USE IOS GPU IN PYTORCH](https://pytorch.org/tutorials/prototype/ios_gpu_workflow.html). Is this what you mean?

Oh, I thought we have a pod for torchvision nightly. NVM, it should be fine.